src/components/ArticleUpdateDate/ArticleUpdateDate.tsx

```diff
@@ -0,0 +1,39 @@
+/** @jsxImportSource react */
+
+import { useEffect, useState } from 'react'
+import classes from './ArticleUpdateDate.module.scss'
+import timeSince from '../../utils/timeSince'
+
+type Props = {
+	path: string
+}
+
+const URL = `https://api.github.com/repos/nowyouknowprogramming/webdev-learning-materials`
+
+const getLastUpdateDate = async (path: string) => {
+	const response = await fetch(`${URL}/commits?path=${path}&per_page=1`)
+	return await response.json()
+}
+
+const ArticleUpdateDate = ({ path }: Props) => {
+	const [updatedAt, setUpdatedAt] = useState<null | string>(null)
+
+	useEffect(() => {
+		getLastUpdateDate(path)
```

Comment:
move the fetching logic to the useCallback

Reply:
I don't understand can you elaborate?

Reply:
Changed to use SWR